Documentation/troubleshooting.rst

```diff
@@ -134,6 +134,104 @@ running agent. Debugging of an individual endpoint can be enabled by running
     CPU 00: MARK 0x7dc2b704 FROM 3978 DEBUG: Handling ICMPv6 type=135
     CPU 00: MARK 0x7dc2b704 FROM 3978 DEBUG: ICMPv6 neighbour soliciation for address b21a8c0:d68a0000
 
+Debugging information
+=====================
+
+``cilium debuginfo`` can print  useful output from the Cilium API. The output
+format is Markdown format so this can be used when reporting a bug on the
+`issue tracker`_.  Running without arguments will print to standard output, but
+you can also redirect to a file like
+
+::
+
+    cilium debuginfo -f debuginfo.md
+
+.. Note::
+
+          Please check the debuginfo file for sensitive information and strip it
+          away before sharing it with us.
+
+Single Node Bugtool
+===================
+
+The ``cilium-bugtool`` captures possibly useful informations about your
+environment for debugging. The tool is meant to be used for debugging a single
+Cilium agent node but in the Kubernetes case if you have multiple Cilium pods
+it can retrieve debugging information from all of them. The tool works by
+collecting command outputs and files from several places and creates archive.
```

Comment:
... works by archiving a collection of command outputs and files from several places.